deps/Makefile

```diff
@@ -263,7 +263,7 @@ endif
 else # NO_GIT
 $2_SRC_DIR = $1-$$($2_BRANCH)
```

Comment:
I suppose so, thanks. What's your opinion, should I rename the checksum files or delete them?